monkey/monkey_island/linux/create_certificate.sh

```diff
@@ -2,8 +2,21 @@
 
 server_root=${1:-"./cc"}
 
+echo "Creating server cetificate. Server root: $server_root"
+# We override the RANDFILE determined by default openssl.cnf
+# This is a known issue with the current version of openssl on Ubuntu 18.04 - once they release
+# a new version, we can delete this command. See
+# https://github.com/openssl/openssl/commit/0f58220973a02248ca5c69db59e615378467b9c8#diff-8ce6aaad88b10ed2b3b4592fd5c8e03a
+# for more details.
+dd bs=1024 count=2 </dev/urandom >~/.rnd
+chmod 666 ~/.rnd
```

Comment:
And if you don't have permissions to ~ or if the file exists?

Reply:
9687b22b

The user which runs ./create_cert will have access to their own ~ folder in enough of the cases that I'm willing to leave this in. No other workaround that doesn't involve carrying openssl with us or messing with worse things in the system that I could fine.